packages/docsearch-react/src/DocSearchModal.tsx

```diff
@@ -27,6 +27,13 @@ export interface DocSearchModalProps extends DocSearchProps {
   onClose?(): void;
 }
 
+const setRealHeight = () => {
```

Comment:
We can move this function inside the effect to remove the dependency problem all together. See "[Moving functions inside effects](https://overreacted.io/a-complete-guide-to-useeffect/#moving-functions-inside-effects)".